src/locations/index.ts

```diff
@@ -54,39 +54,59 @@ export async function getLocations(options: LocationCmdOptions) {
 }
 
 export function formatLocations(locations: Array<LocationMetadata>) {
-  const formatted: Array<string> = [];
+  const publicLocations: Array<string> = [];
```

Comment:
I dont understand the use of this split given we already append `private` keyword for all private locations. 

If the purpose of this is to display them separately, then please change the grouping logic and rendering logic to account for empty private location array.